.hintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
- [ ] You don't have a link to your issues page of your repository. Kindly add a link to your issues page in your readme file to make it descriptive 😊

Reply:
Thanks done

Reply:
It's great on working on the changes 👍 Inshallah next reviewer will approve your project 😊